packages/manager/apps/pci-block-storage/src/api/select/catalog.ts

```diff
@@ -326,3 +329,48 @@ export const mapFilterLeastPrice = <T extends TCatalogGroup>(
     },
   };
 };
+
+export type TModelPreselection = {
+  isPreselected: boolean;
+};
+
+export const mapPreselection = (type?: string | null) => <
+  T extends TVolumeAddon
+>(
+  model: T,
+) => {
+  const possibleTechnicalName = model.pricings.flatMap((p) => p.specs.name);
```

Comment:
```suggestion
  const possibleTechnicalName = model.pricings.map((p) => p.specs.name);
```
name isn't an array isn't it ?